### ISSUE TITLE
Pet steal experience from necromancers

### ISSUE THREAD

**Issue by Lilithelle** [torilmud/issues#457]
I don't care about the salty old users that can handle this, but I've seen a few new players that start off as necromancers and hit this issue immediately.  For new players, who have mudded elsewhere, this seems to take them by surprise.  Given the power of other classes, like the warrior sub-classes, I'm wondering if this isn't something we can consign to the past.  Or at least at lower levels to help new players.

**Comment by Dartan82** [comment]
Yes in the other MUDs I've played, pets are given 0 exp and are an extension of the user.  All MMOs do this these days.

**Comment by ghost** [comment]
given that non player non pets no longer gain xp, i'm aligned with this, no pet should ever gain xp while grouped, having pets grouped is the ideal method of checking their hp and casting things like group heal on them, you took away mobs gaining xp, now do the same with pets

**Comment by Inames** [comment]
Absolutely agree with this, it's always irritated me that using your class abilities/features could have a negative influence on your leveling experience.
